virtualization/windowscontainers/management/container_networking.md

```diff
@@ -235,13 +266,58 @@ C:\> docker run -it --network=MyTransparentNet --ip=10.80.123.32 windowsserverco
 
 ## Caveats and Gotchas
 
 ### Firewall
 
 The container host requires specific Firewall rules to be created to enable ICMP (Ping) and DHCP. ICMP and DHCP are required by Windows Server Containers to ping between two containers on the same host, and to receive dynamically assigned IP addresses through DHCP. In TP5, these rules will be created through the Install-ContainerHost.ps1 script. Post-TP5, these rules will be created automatically. All Firewall rules corresponding to NAT port forwarding rules will be created automatically and cleaned up when the container stops.
 
+### Existing vSwitch Blocking Transparent Network Creation
+
+When a transparent network is created, it is bound to the external network by a vSwitch which is in turn connected to an external network adapter on the container host. The vSwitch for the transparent network is either created or discovered (if pre-existing) by Docker. The vSwitch is then connected to the external network when the Windows Host Network Service (HNS) identifies the network adapter on the host with which to bind. With this in mind, *it is possible to run into problems with connecting the vSwitch to the host's external network adapter if you already have an external vSwitch created that is not visible to Docker.* 
+
+For example, this issue would arise if you were to first create a new vSwitch on your host while the Docker service was running, then try to create a transparent network. In this case, Docker would not recognize the switch that you already created, so it would create a new vSwitch for the transparent network. The HNS service would then be unable to bind the new switch (created by Docker) to the host's external network adapter, because the adapter would already be connected to the first vSwitch that you created (the switch created 'out-of-band').
+
+There are two possible solutions for this issue. 
+
+First, you can of course delete the vSwitch that was created out-of-band, which will allow docker to create a new vSwitch and connect it to the host network adapter without issue. Before choosing this approach, ensure that your out-of-band vSwitch is not being used by other services (e.g. Hyper-V).
+
+Alternatively, if you decide to use an external vSwitch that was created out-of-band, restart the Docker and HNS services to *make the switch visible to Docker* (this cause Docker to connect to the existing switch rather than creating a new one).
```

Comment:
A third option is if they have a second network adapter to which they can bind the Transparent / External Vswitch to. This is listed as an option in the article above, "-o ..."